client/payment-details/transaction-breakdown/fees-breakdown/index.tsx

```diff
@@ -69,19 +101,14 @@ const FeesBreakdown: React.FC< {
 		currency,
 		isDiscounted,
 		displayFixedPart,
-	}: {
-		type: string;
-		additionalType?: string;
-		percentage: number;
-		fixed: number;
-		currency: string;
-		isDiscounted?: boolean;
-		displayFixedPart?: boolean;
-	} ) => {
+		taxInfo,
+		isTaxRow,
```

Comment:
Where is the `isTaxRow` provided to the `FeeRow` component?

I see `FeeRow` forwards it to the `BreakdownFeeRate`. But I don't see it provided by the consumers of `FeeRow`.

Reply:
I added that prop initially, but then I changed the approach and added a dedicated `TaxFeeRow` component so this prop is not needed. I removed it in this commit [22f1907](https://github.com/Automattic/woocommerce-payments/pull/10707/commits/22f19073e4071ce72200ed4922b554a41c64aa47). And also took the chance to move some components to its own file, hoping to improve organization a bit. Thanks for catching it!